Scripts/Enemy/NoobEnemy.cs

```diff
@@ -0,0 +1,127 @@
+using Godot;
+
+public class NoobEnemy : Enemy
+{
+    [Export]
+    public float AttackCooldown = 3f;
+    [Export]
+    public float AttackDuration = 0;
+    [Export]
+    public float ChargeSpeed = 60f;
+    [Export]
+    public float ChargeDistance = 10f;
+    [Export]
+    public float DetectDistance = 8f;
+    [Export]
+    public float AttackDamage = 2f;
+    [Export]
+    public float KnockbackFactor = 0.4f;
+
+    private SpatialMaterial material;
+    private float attackCooldownLeft = 3f;
+    private float attackDurationLeft = 0f;
+
+    public NoobEnemy()
+    {
+        MaxHealth = 100;
+        acceleration = 20f;
+    }
+
+    public override void _Ready()
+    {
+        base._Ready();
+        attackCooldownLeft = AttackCooldown;
+        material = new SpatialMaterial()
+        {
+            AlbedoColor = Colors.White,
+            FlagsTransparent = true
+        };
+
+        // CSGCylinder cylinder = GetNode<CSGCylinder>("CSGCylinder");

```

Comment:
Oops forgot to remove this. Want me to remove?

Reply:
ya why not